pkg/utils/log.go

```diff
@@ -74,6 +74,8 @@ func NewLogger(prefix string, level LogLevel) *Logger {
 		l.level = level
 	}
 
+	stdout, _ = os.OpenFile("/var/log/heketi/heketi.log", os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
```

Comment:
@ramkrsna Logging to a file from a long-running service is not as easy as simply opening it for creation or appending and then writing to it: this way the log file will never get rotated and grow indefinitely. One usually needs some kind of signal handler to reopen the file (e.g on USR2). This way a logrotate system can work.

Logging to stderr/stdout or syslog circumvents these complications.